tests/views/test_collection_exercise.py

```diff
@@ -2858,6 +2858,14 @@ def test_view_sample_ci_summary(
         )
         self.assertIn("Return to 000000 Collection exercise".encode(), response.data)
         self.assertIn("Choose a version".encode(), response.data)
+        self.assertIn(
+            "A collection instrument is only created once both an EQ formtype and a CIR".encode(),
+            response.data,
+        )
+        self.assertIn(
+            "version are selected.<br>Each formtype requires a CIR version.".encode(),
+            response.data,
+        )
```

Comment:
Going forward we don't need to test pure content on a page like this. Using it as verification that we are on a certain page or it's functionally working is where we should stop